framework/src/meshgenerators/SideSetsFromNodeSetsGenerator.C

```diff
@@ -16,16 +16,23 @@ registerMooseObject("MooseApp", SideSetsFromNodeSetsGenerator);
 InputParameters
 SideSetsFromNodeSetsGenerator::validParams()
 {
-  InputParameters params = MeshGenerator::validParams();
+  InputParameters params = SideSetsGeneratorBase::validParams();
+
+  params.suppressParameter<Point>("normal");
+  params.suppressParameter<Real>("normal_tol");
+  params.suppressParameter<bool>("fixed_normal");
+  params.suppressParameter<bool>("replace");
+  params.suppressParameter<bool>("include_only_external_sides");
+  params.suppressParameter<std::vector<BoundaryName>>("included_boundaries");
+  params.suppressParameter<std::vector<SubdomainName>>("included_subdomains");
+  params.suppressParameter<std::vector<SubdomainName>>("included_neighbors");
```

Comment:
yeah this is not good. You were right initially, does not make sense for this one in its current implementation

Reply:
I dont see us implementing the rest of the parameters

Reply:
so let's roll back on this one